src/main/java/world/bentobox/chat/listeners/ChatListener.java

```diff
@@ -27,14 +29,14 @@
 
     private final Chat addon;
     private final Set<UUID> teamChatUsers;
-    private final Set<Island> islands;
+    private final HashMap<Island, Set<Player>> islands;
```

Comment:
`private final Map<Island, Set<Player>> islands;`

Reply:
Thanks, I changed it.